README.md

```diff
@@ -423,7 +423,7 @@ message_bus also supports an in-memory backend. This can be used for testing or
 MessageBus.configure(backend: :memory)
 ```
 
-The `:clear_every` option supported by the PostgreSQL backend is also supported by the in-memory backend.
+The `:clear_every` option is also supported by the in-memory backend.
```

Comment:
Good point - just pushed some README changes :+1:

Reply:
Hm, now it's only documented in the redis section. Maybe it should be moved into a new 3rd-level section, e.g. after `### Keepalive`?

Reply:
I think it's gonna be confusing to try and pull it out into its own section, because the different backends recommend setting the configuration in different ways. Instead, I've added a dedicated description in the Redis section.